src/quacc/recipes/espresso/core.py

```diff
@@ -0,0 +1,200 @@
+"""Core recipes for espresso."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase import Atoms
+
+from quacc import SETTINGS, job
+from quacc.calculators.espresso.espresso import (Espresso, EspressoProfile,
+                                                 EspressoTemplate)
+from quacc.runners.ase import run_calc
+from quacc.schemas.ase import summarize_run
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from quacc.schemas._aliases.ase import RunSchema
+
+
+@job
+def static_job(
+    atoms: Atoms,
+    preset: str | None = None,
+    copy_files: list[str] | None = None,
+    tmpdir: str | None = None,
+    parallel_info: dict[str] | None = None,
+    calc_clean: bool = True,
```

Comment:
Here and in several places, docstrings will need to be added.

After our discussion, I'm not sure we'll need `tmpdir` or `calc_clean`. Let's circle back to those.